UPGRADE-1.11.md

```diff
@@ -4,3 +4,13 @@ doctrine/cache will no longer be maintained and all cache implementations have
 been marked as deprecated. These implementations will be removed in 2.0, which
 will only contain interfaces to provide a lightweight package for backward
 compatibility.
+
+There are two new classes to use in the `Doctrine\Common\Cache\Psr6` namespace:
+* The `CacheAdapter` class allows using any Doctrine Cache as PSR-6 cache. This
+  is useful to provide a forward compatibility layer in libraries that accept
+  Doctrine cache implementations and switch to PSR-6.
+* The `DoctrineProvider` class allows using any PSR-6 cache as Doctrine cache.
+  This implementation is designed for libraries that leak the cache and want to
+  switch to allowing PSR-6 implementations. This class is marked as internal and
```

Comment:
Isn't "internal" in conflict with the previous line, stating that this is for libraries?
What about `@deprecated` instead?

Reply:
It's mainly for internal libraries - I'm not sure how many non-doctrine libraries support injecting doctrine/cache implementations and leak those to the outside world. I wanted to add some kind of designation to avoid people thinking that this is code that will be maintained or should be used outside of compatibility layers, but I can also drop this to avoid triggering deprecation notices or anything else (e.g. when using the Symfony Debug class loader).

Reply:
the debug class loader will complain only if ppl extend the class, not if they just use it
but I think removing the annotation is fine also. The full component is sunsetted anyway, so the message should be sent not only for this class but for all of them in the lib.

Reply:
I removed the designation as technically doctrine/persistence would be violating it already. If people want to shoot themselves in the foot with this, I won't complain.